### ISSUE TITLE
Add header and footer to default base template

### ISSUE THREAD

**Issue by john0isaac** [render-engine/render-engine#609]
Render engine has a default base.html template that gets rendered if you don't have a _base.html template

currently it's missing the html header and footer tags.
Need to add them to match _base.html template

Cc: @kjaymiller

**Comment by Mahhheshh** [comment]
The base.html extends all the HTML from `base_templates/_base.html` so there is no need to explicitly define the headers and footer in the `base.html` file.

**Comment by kjaymiller** [comment]
@Mahhheshh is correct. 

This is the components of `_base.html`

```
  <body {% if body_class is defined %}class="{{"".join(body_class)}}"{%endif%}>
        {% block header %}{% endblock %}
        {% block page_title %}{% endblock %}
        {% block content %}{% endblock %}
        {% block footer %}
        {% endblock %}
    </body>
```

We may need to check the template used on `render_engine_init` to make sure that it isn't overriding the base.

**Comment by kjaymiller** [comment]
@all-contributors please add @john0isaac for Maintenance UserTesting
@Mahhheshh maintenance

**Comment by kjaymiller** [comment]
@all-contributors please add @Mahhheshh for maintenance

**Comment by kjaymiller** [comment]
Looking at the base template from - https://github.com/render-engine/cookiecutter-render-engine-site/blob/main/%7B%7Bcookiecutter.project_slug%7D%7D/templates/_base.html

It seems like it is setup as expected with references to header and footer.

**Comment by john0isaac** [comment]
@kjaymiller I think you forgot about this issue it wasn't with the cookie cutter base template.
You told me to remove the _base.html and render engine will use the default base.html.

the problem is that the default base.html is what's missing the header and the footer as I have explained not the cookiecutter template.

**Comment by kjaymiller** [comment]
You're correct. Interesting enough I think there are some things that we cxan do if we plan this out a bit.

I'm sure we can plan this out and get to a good solution.
